libc/newhdrgen/header2.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python
+#
+# ===- HeaderFile Class for libc function headers  -----------*- python -*--==#
```

Comment:
this file should have a name and license header that explain that it's for the GPU `--export-decls` version of headers.